doc/doxygen/buildfiles/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2018 Pieter Conradie <https://piconomix.com>
```

Comment:
What is this license file for?

Reply:
I was looking for a nice doxygen customization (haven't done much of that before, usually used the default) and I ended up borrowing the doxygen theme from this project -> https://github.com/piconomix/doxygen-github-markdown-theme

I'm treating this whole CSS as a starting point really, and will be working on making the doxygen output more consistent with the look-and-feel of the ScummVM website.

I don't have much experience with licenses and I'm not sure if this should be kept, considering that we won't be using this as-is.